### DISCUSSION TITLE
Impossible de supprimer un listener via l'ep : https://sb-api.paygreen.fr/notifications/listeners

### DISCUSSION THREAD

**Discussion by aminemts** [PayGreen/documentation#24] [Q&A] [answered]
Bonjour, je n'arrive pas à supprimer les listeners précedement crée.
Avant j'avais la possibilité de les supprimé en appellant l'ep https://sb-api.paygreen.fr/notifications/listeners/lis_{id}.
Mais maintenant ca me retourne  la response : 404 (not found).
Or quand je fais un get via : https://sb-api.paygreen.fr/notifications/listeners , j'ai bien la lister des listeners
Merci pour votre retour.

**Comment by gaelDolle** [comment, marked as answer]
Bonjour,

La route appelée est bonne, merci de vérifier votre intégration.
(Le "lis_" fait parti intégrante de l'id du listener)

Cordialement
